bin/migrate-goldens.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+import { join, normalize, sep } from 'path';
+import { mkdir, rename } from 'node:fs/promises';
+import commandLineArgs from 'command-line-args';
+import { glob } from 'glob';
+import { stdout } from 'node:process';
+
+const { pattern = './test/**' } = commandLineArgs({ name: 'pattern', type: String, defaultOption: true }, { partial: true });
+const oldSuffix = 'screenshots/ci/golden';
```

Comment:
For later: I want to revisit all these directory names -- `ci`, `current` & `golden`. But should be easy to adjust this later if we change them.